Podcast/EpisodeDetailHeaderViewCell.swift

```diff
@@ -94,7 +95,8 @@ class EpisodeDetailHeaderViewCell: UITableViewCell {
         addSubview(episodeTitleLabel)
         
         dateLabel = UILabel(frame: CGRect(x: marginSpacing, y: 0, width: frame.width - 2 * marginSpacing, height: 0))
-        dateLabel.font = UIFont.systemFont(ofSize: 14)
+        dateLabel.font = UIFont.systemFont(ofSize: 12)
+        dateLabel.textColor = UIColor.colorFromCode(0x959ba5)
```

Comment:
put this in utilities! it might already be in there .. if not make it named podcast[ColorDescription]